spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -771,6 +783,10 @@ describe('Redfish Systems Root', function () {
     });
 
     it('should return a valid system', function() {
```

Comment:
A suggestion for you, use describe to group all test for your API.
We have two many test in this file and should divide them into small groups.